src/components/Header.astro

```diff
@@ -0,0 +1,42 @@
+---
+import { Icon } from "astro-icon"
+import AstroLogo from "../components/AstroLogo.astro"
+import { mainSiteLink, socialLinks } from "../data/links"
+---
+
+<header
+	class="noise-container flex h-24 items-center gap-6 border-b border-astro-gray-500 px-8"
+>
+	<div class="noise"></div>
+	<a href="/" class="transition hover:opacity-75" title="Home">
```

Comment:
Icon links like this should use `aria-hidden="true"` on the icon itself. Applies here and to the Twitter, Discord, and GH icons in the top bar.